core/src/test/scala/unit/kafka/admin/ConfigCommandTest.scala

```diff
@@ -479,6 +460,9 @@ class ConfigCommandTest extends QuorumTestHarness with Logging {
       "--alter",
       "--add-config", "a=b,c=d"))
 
+    val zkClient = mock(classOf[KafkaZkClient])
+    when(zkClient.getEntityConfigs(anyString(), anyString())).thenReturn(new Properties())
```

Comment:
note: Using mock `KafkaZkClient` is fine here because we just want to test the updated config is correctly parsed by zkClient. Same as below. Thanks.